src/elements/common/interactionTargets.js

```diff
@@ -40,6 +40,12 @@ export const ACTIVITY_TARGETS = {
     TASK_APPROVE: 'activityfeed-taskapprove',
     TASK_COMPLETE: 'activityfeed-taskcomplete',
     TASK_REJECT: 'activityfeed-taskreject',
+    TASK_OPTIONS: 'activityfeedtaskoptions', // component
+    TASK_OPTIONS_DELETE: 'activityfeed-taskoptions-delete',
+    TASK_OPTIONS_EDIT: 'activityfeed-taskoptions-edit',
+    COMMENT_OPTIONS: 'activityfeedcommentoptions', // component
```

Comment:
Is there a typo? `activityfeed-commentoptions`? Should we also pull it out into something like `ACTIVITY_COMPONENTS`?

Reply:
not allowed to have `-` in `component`